misc/save_new_version.py

```diff
@@ -0,0 +1,28 @@
+import os
+from datetime import date
+
+import requests
+
+version = os.getenv('RELEASE_VERSION')
+version = version[1:]
+version_num = version.split('.')
+major = int(version_num[0])
+minor = int(version_num[1])
+patch = int(version_num[2])
+release_date = date.today().strftime('%Y-%m-%d')
+
+payload = {
+    'version': version,
+    'major': major,
+    'minor': minor,
+    'patch': patch,
+    'date': release_date
+}
+
+username = os.getenv('METADATA_USERNAME')
+password = os.getenv('METADATA_PASSWORD')
+
+response = requests.post('http://54.90.48.192/add_version/main',
```

Comment:
How about making the IP address (i.e. `http://54.90.48.192`) an ENV VAR (e.g. `METADATA_SERVICE_URL`) as well so later if we update it to something like HTTPS://DOMAIN we don't need to change the code again? Same for the other place.

Reply:
Side note: could you add all newly created credentials (username, password, url, etc.) to Vault? (and we may let Action authenticate with Vault in the future) Thanks!

Reply:
Added to vault

Reply:
> How about making the IP address (i.e. `http://54.90.48.192`) an ENV VAR (e.g. `METADATA_SERVICE_URL`) as well so later if we update it to something like [HTTPS://DOMAIN](HTTPS://DOMAIN) we don't need to change the code again? Same for the other place.

Good point.